src/mmrelay/matrix/sync_bootstrap.py

```
@@ -0,0 +1,1199 @@
+                                nio_responses.SyncResponse._get_invite_state = (
+                                    original_descriptor
+                                )
+                            else:
+                                if (
+                                    "_get_invite_state"
+                                    in nio_responses.SyncResponse.__dict__
+                                ):
+                                    try:
+                                        del nio_responses.SyncResponse._get_invite_state
+                                    except (AttributeError, TypeError):
+                                        pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(AttributeError, TypeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(AttributeError, TypeError): ...`

(SIM105)